src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -358,7 +358,12 @@ protected function validateDateFormat($attribute, $value, $parameters)
             return false;
         }
 
-        $date = DateTime::createFromFormat($parameters[0], $value);
+        if (!preg_match('/v/', $parameters[0])) {
+            $date = DateTime::createFromFormat($parameters[0], $value);
+        } else {
+            $dateString = date($parameters[0], strtotime($value));
+            $date = new DateTime($dateString);
+        }
```

Comment:
@tillkruss Thanks a lot for your feedback - most appreciated! I've updated the code to reflect your feedback.